helm/karpenter/templates/node-renewer.yaml

```diff
@@ -56,15 +56,16 @@ metadata:
   name: {{ include "karpenter.serviceAccountName" . }}-renewer
   labels:
     {{- include "karpenter.labels" . | nindent 4 }}
+    app.kubernetes.io/instance: karpenter-renewer
```

Comment:
Wouldn't using [the selector labels](https://github.com/giantswarm/karpenter-app/blob/main/helm/karpenter/templates/_helpers.tpl#L36-L39) fix the issue?